resources/lib/menu_functions.py

```diff
@@ -1030,7 +1030,7 @@ def display_movies_type(menu_params, view):
     params["Limit"] = item_limit
     path = get_jellyfin_url("/Users/{userid}/Items", params)
     url = sys.argv[0] + "?url=" + quote(path) + "&mode=GET_CONTENT&media_type=movies&sort=none"
-    add_menu_directory_item('{}{} ({})'.format(view_name, translate_string(30349), item_limit), url)
+    add_menu_directory_item(('{}{} ' + get_filtered_items_count_text()).format(view_name, translate_string(30349)), url)
```

Comment:
I personally don't like this formatting.  Mixing string concatenation methods like this seems arbitrarily confusing compared to leaving the placeholders and doing it as an argument like the other variables here.
```suggestion
    add_menu_directory_item('{}{}{}'.format(view_name, translate_string(30349), get_filtered_items_count_text()), url)
```
Also, the `get_filtered_items_count_text()` already includes a space at the front of the string, so including it here as well results in a double space